clierrors/errors.go

```diff
@@ -0,0 +1,17 @@
+package clierrors
+
+var (
+	ErrInvalidUpdate = "Invalid state passed. Specify either activate or archive\n"
```

Comment:
nit: maybe give this a more specific name since it references state exclusively?

Reply:
Done.